lib/experimental/fonts/font-library/font-library.php

```diff
@@ -134,7 +134,8 @@ function wp_unregister_font_collection( $collection_id ) {
 	'slug'        => 'default-font-collection',
 	'name'        => 'Google Fonts',
 	'description' => __( 'Add from Google Fonts. Fonts are copied to and served from your site.', 'gutenberg' ),
-	'src'         => 'https://s.w.org/images/fonts/16.7/collections/google-fonts-with-preview.json',
+	// TODO: This URL needs to be updated to the wporg hosted one prior to the Gutenberg 17.6 release.
+	'src'         => 'https://raw.githubusercontent.com/WordPress/google-fonts-to-wordpress-collection/main/releases/gutenberg-17.6/google-fonts.json',
 );
 
 wp_register_font_collection( $default_font_collection );
```

Comment:
Do we need the font collection registered on every request? If the font collection is only needed in wp-admin and rest api requests, maybe there is a hook we can for collection registration.